src/sentry/models/organizationmembermapping.py

```diff
@@ -21,6 +21,9 @@ class OrganizationMemberMapping(Model):
     __include_in_export__ = False
 
     organization_id = HybridCloudForeignKey("sentry.Organization", on_delete="CASCADE")
+    # TODO: allow null values for organizationmember_id column. We will later repair and populate these columns with
+    # values; and remove null=True.
+    organizationmember_id = BoundedBigIntegerField(db_index=True, null=True)
```

Comment:
Should this be a `HybridCloudForeignKey`?

Reply:
This cannot be `HybridCloudForeignKey` as the org member ids are not unique across regions.

Reply:
Should we make them snowflake ids or something? Not sure what the general process is there

Reply:
@wedamija that's one approach. @markstory has suggested making use of outbox messages to propagate deletions rather than tombstones; which will be easier to do.

Reply:
Changing to snowflakes is an option here. I think we can use outbox to propagate the deletes here, if that turns out to be clunky then we can go to snowflakes.